nimx/naketools.nim

```diff
@@ -926,10 +894,14 @@ proc getConnectedAndroidDevices*(b: Builder): seq[string] =
 
 proc installAppOnConnectedDevice(b: Builder, devId: string) =
     let conf = if b.debugMode: "debug" else: "release"
-    var apkPath = b.buildRoot / b.javaPackageId / "build" / "outputs" / "apk" / b.javaPackageId & "-" & conf & ".apk"
-    if not b.useGradle:
-        apkPath = b.buildRoot / b.javaPackageId / "bin" / b.appName & "-" & conf & ".apk"
+    var apkPath = b.buildRoot / b.javaPackageId / "build" / "outputs" / "apk" / conf / b.javaPackageId & "-" & conf & ".apk"
+
     direShell b.adbExe, "-H", b.adbServerName, "-s", devId, "install", "-r", apkPath
+    #[
+        # start activity, TODO: prevent this on CI's
+        var activityName =  b.javaPackageId & "/" & b.activityClassName
+        direShell b.adbExe, "shell", "am", "start", "-n", activityName
```

Comment:
Why is this gone?

Reply:
This is new feature, I use this in my local fork. But looks like we need to do this in other way, cuz we can have problems on CI.